### ISSUE TITLE
How do I use this?

### ISSUE THREAD

**Issue by kasetg** [UnquietCode/Tumblr-Likes-Grid#5]
I have no idea how to, and I really want it.

**Comment by UnquietCode** [comment]
Yeah, admittedly I didn't really leave instructions on how to run it locally, but it will involve some level of developer chops to set up a PHP server and get the OAuth piece working. Ideally someone would port this to use a front end library like OAuth.io to avoid requiring the server, and then I could just host it myself again.

**Comment by UnquietCode** [comment]
@kasetg see #6 and the related bounty: https://www.bountysource.com/issues/15132434

**Comment by leocavallini** [comment]
Hello gentlemen,
I'm trying to run this code on my server but it's not working. I guess coffee-script.js is missing for example, although it doesn't appear as a requested file on Chrome console log. I can successfuly login but it keeps loading and my Tumblr likes don't appear.
Besides, I can host what's needed "forever" to leave the code up for anyone to use. I'm photographer and need my website up all the time.
Thanks!

**Comment by leocavallini** [comment]
Actually I saw the bounty.
I wasn't familiar to this and honestly I'm also new to handling GitHub stuff as a logged user, I just downloaded some stuff like FileZilla etc. I was a web designer before becoming photographer and still develop my personal projects like my blog and portfolio but those two guys are kinda new to me.
So, the bounty... I see pyk has worked this out and it also works for me. If you need me, my server is hosted on mediatemple and is available to host anything needed.
Cheers!
